MetaMask/metamask-extension
# update contract-metadata to 2.1.0

## Explanation
Updates to latest version of contract metadata, bumping a major version. The only breaking change is dropping support for node v12 which the extension does not rely upon.

## Manual Testing Steps
1. See https://github.com/MetaMask/contract-metadata/releases/tag/v2.1.0 and use the provided changelog to check if token addresses are validated in the add token flow.

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
